server/services/case-service.js

```diff
@@ -34,9 +34,16 @@ const createCaseService = (apiUrl) => {
       apiUrlBuilder.searchParams.append('VERSION2', 'true')
       apiUrlBuilder.searchParams.append('page', page || '1')
       apiUrlBuilder.searchParams.append('limit', limit)
+
       if (subsection === 'added') {
         apiUrlBuilder.searchParams.append('recentlyAdded', 'true')
       }
+      if (subsection === false || subsection === null || subsection === undefined) {
```

Comment:
Does this need a flag? cos what happens if you turn off the outcomes flag?

Reply:
The `getPagedCaseList` method is already behind a feature flag `ENABLE_SERVERSIDE_PAGING`, but yes I suppose I need to take into account if this flag is enabled but `ENABLE_HEARING_OUTCOMES` isn't.